keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ArtifactVersionDeploying.kt

```diff
@@ -0,0 +1,10 @@
+package com.netflix.spinnaker.keel.events
+
+/**
+ * An event fired when we launch a task to deploy a new version to a cluster
+ */
+data class ArtifactVersionDeploying(
+  val resourceId: String,
+  val artifactVersion: String,
+  val provider: String = "aws"
```

Comment:
Why default this? I don't think we should have a preference for providers. 😄

Reply:
ehh, was just copying from elsewhere. I can remove

Reply:
Actually I don't even need this field now. I'll just remove it!